src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -23,8 +25,15 @@ <h4>{{ 'artemisApp.examManagement.editExam' | artemisTranslate }}</h4>
                     <input class="form-control" required type="text" name="title" id="title" [(ngModel)]="exam.title" />
                 </div>
             </div>
-
-            <div class="row">
+            <!-- Creation of TestExams is currently hidden for all non-admins, as the feature is not yet fully implemented -->
+            <div class="form-group" *ngIf="isAdmin">
+                <div>
+                    <label class="form-check-label" jhiTranslate="artemisApp.examManagement.testExam.examMode">Exam Mode</label>
```

Comment:
Might not be in scope of this PR, but is there a particular reason why we are using both jhiTranslate as well as the artemis translate pipe in this file?

Reply:
I have changed the subheaders to `jhiTranslate`. 
Some modules (like the `jhi-date-time-picker` don't have a build-in translate pipe and therefore need a String as Input-Value. Also, if a dynamic changing of labels is necessary, the pipe was used to provide the suitable String directly to the component / user (e,g, `Start of working time` (RealExam) and `Start of working window` (TestExam) for the input-field of the startDate